### PR TITLE
Email content

### PR DESCRIPTION
**Pull request by harisibrahimkv**

### CODE REVIEW

**Comment by vnbang2003** [comment]
@harisibrahimkv Can we even make text changes in txt files also . Some email client might not support HTML emails in that case txt file will work.

**Comment by harisibrahimkv** [comment]
@vnbang2003 Made changes to txt files as well.

Please verify the content once. I am not completely sure about the context of the emails - who is it being sent to and when. I've written the email content based on my understanding.

**Comment by harisibrahimkv** [comment]
Also, all the files need to be passed in proper context values and links. I've just put placeholders for everything at the moment.

File: wye/templates/email_messages/organisation/additional_member.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Hey {{full_name}},
+
+You've added {{ new_member_name }} to your organization. {{ new_member_name }} can now access your organization dashboard and manage workshops. Visit your profile (link-to-profile) to view or update your organization details.
+
```
Comment (vnbang2003) [inline wye/templates/email_messages/organisation/additional_member.txt]: Not able to understand profile reference ?

Comment (harisibrahimkv) [inline wye/templates/email_messages/organisation/additional_member.txt]: Doesn't a user have a profile page?

Comment (harisibrahimkv) [inline wye/templates/email_messages/organisation/additional_member.txt]: @vnbang2003 Sorry, that profile refers to the organization's profile page. When a new member is added to an organization, he/she can edit the organization details right?

Comment (vnbang2003) [inline wye/templates/email_messages/organisation/additional_member.txt]: Profile page doesn't update the organisaiton update .

Comment (vnbang2003) [inline wye/templates/email_messages/organisation/additional_member.txt]: can you rename variable link-to-profile to organization list page

**Comment by harisibrahimkv** [comment]
@vnbang2003 Renamed.

**Comment by vnbang2003** [comment]
Can you help with content for #11

**Pull request merged**